extension_cpp/csrc/muladd.cpp

```diff
@@ -1,7 +1,28 @@
-#include <torch/extension.h>
+#include <Python.h>
+#include <ATen/Operators.h>
+#include <torch/all.h>
+#include <torch/library.h>
 
 #include <vector>
 
+extern "C" {
+  /* Creates a dummy empty _C module that can be imported from Python.
+     The import from Python will load the .so associated with this extension
+     built from this file, so that all the TORCH_LIBRARY calls below are run.*/
```

Comment:
I'm not sure about the technical details of how the import _C will "load the .so". Is that a Python import module law for extensions? Or is it that by calling a method from this file, a side effect is to load the .so that the method lives in, which will by side effect run the TORCH_LIBRARY calls below?